src/commands/Moderation/multiban.ts

```diff
@@ -28,93 +28,103 @@ export default class extends Command {
 	}
 
 	public async run(context: CommandContext, args: string[]): Promise<void | Message> {
-		const targets: RegExpMatchArray = args.slice(0).join(" ").match(UserMentionAndID) ?? [];
-		let reason: string = args.slice(targets.length).join(" ").replace(UserMentionAndID, "");
+		const potentialTargets: RegExpMatchArray = <RegExpMatchArray>args.slice(0).join(" ").match(UserMentionAndID);
+		let validTargets: (GuildMember | User)[] = [];
+		let invalidTargets: number = 0;
+		let fullList: string = "";
+		let reason: string = args.slice(potentialTargets?.length).join(" ").replace(UserMentionAndID, "");
 
 		if (reason === "") reason = await this.client.bulbutils.translate("global_no_reason", context.guild?.id, {});
-		let fullList: string = "";
 
-		if (targets.length <= 1) {
+		for (const potentialTarget of potentialTargets) {
+			const t = potentialTarget.replace(NonDigits, "");
+			if (!t.length) continue;
+
+			let target: User | GuildMember | null = t ? <GuildMember>await context.guild?.members.fetch(t).catch(() => null) : null;
+
+			if (!target) {
+				try {
+					target = await this.client.users.fetch(t);
+				} catch (error) {
+					invalidTargets++;
+					continue;
+				}
+			}
+
+			if (target instanceof GuildMember)
+				if (await this.client.bulbutils.resolveUserHandle(context, await this.client.bulbutils.checkUser(context, target), target.user)) continue;
+
+			validTargets = [...validTargets, target];
```

Comment:
```suggestion
			validTargets.push(target);
```
Creating a new array with the spread operator is harder to understand and is likely slower. Use `Array#push` instead

Reply:
Fixed in 4216fa7